robot-core/src/main/java/org/obolibrary/robot/TemplateOperation.java

```diff
@@ -144,9 +146,9 @@ public static OWLDatatype getDatatype(QuotedEntityChecker checker, String name)
    * @return a new annotation with property and string literal value
    * @throws Exception if the annotation property cannot be found
    */
-  public static OWLAnnotation getStringAnnotation(
-      QuotedEntityChecker checker, String template, String value) throws Exception {
-    return TemplateHelper.getStringAnnotation(checker, template, value);
+  public static void getStringAnnotation(QuotedEntityChecker checker, String template, String value)
```

Comment:
It doesn't make sense to me that these signatures are changed to `void`, even if they're deprecated.

Reply:
I think that was an oversight on my part - changing these back.